integration_tests/test_fee_payer.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import json
+
+from .utils import sign_single_tx_with_options
+
+pytestmark = pytest.mark.normal
+
+
+def test_different_fee_payer(cluster, tmp_path):
+    transaction_coins = 100
+    fee_coins = 1
+
+    receiver_addr = cluster.address("community")
+    sender_addr = cluster.address("signer1")
+    fee_payer_addr = cluster.address("signer2")
+
+    unsigned_tx_txt = tmp_path / "unsigned_tx.txt"
+    partial_sign_txt = tmp_path / "partial_sign.txt"
+    signed_txt = tmp_path / "signed.txt"
+
+    receiver_balance = cluster.balance(receiver_addr)
+    sender_balance = cluster.balance(sender_addr)
+    fee_payer_balance = cluster.balance(fee_payer_addr)
+
+    unsigned_tx_msg = cluster.transfer(
+        sender_addr,
+        receiver_addr,
+        "%sbasecro" % transaction_coins,
+        generate_only=True,
+        fees="%sbasecro" % fee_coins,
+    )
+
+    json.dump(unsigned_tx_msg, unsigned_tx_txt.open("w"))
```

Comment:
Could have buffered file writing here, since the file object might not be closed timely, depend on gc.